tests/foreman/cli/test_discoveryrule.py

```diff
@@ -29,12 +29,8 @@
 from requests import HTTPError
 
 from robottelo.cli.base import CLIReturnCodeError
-from robottelo.cli.discoveryrule import DiscoveryRule
 from robottelo.cli.factory import CLIFactoryError
```

Comment:
We are going to remove cli.factory and going to replace with cli_factory. You can use cli_factory for Error Codes here.

Reply:
updated

Reply:
I tried to update it and the PRT is failing looks like some issue with the import , not investigated deeply,  we can do it with the original code movement anyway.

Reply:
No issues.